core/trino-main/src/main/java/io/trino/sql/planner/ExpressionInterpreter.java

```diff
@@ -1271,29 +1267,6 @@ public Object visitCast(Cast node, Object context)
             }
         }
 
-        @Override
-        protected Object visitArray(Array node, Object context)
```

Comment:
Dear Revewier,

the default (super's) impl delegates to visitExpression, which throws:
https://github.com/trinodb/trino/blob/7a6499fc0774f3745ebd4835828e26d4c8252018/core/trino-main/src/main/java/io/trino/sql/planner/ExpressionInterpreter.java#L1575